### ISSUE TITLE
Any updates with scroll locking?

### ISSUE THREAD

**Issue by doomsayer13** [benkeen/ace-diff#21]
@benkeen pls, do not let the project die! :)

**Comment by benkeen** [comment]
No, no update, sorry! I've returned to work on another project and this one kind of fell by the wayside... sorry, @doomsayer13! I do hope to get back to it at some point but time will tell.

**Comment by jmls** [comment]
I was looking for scroll locking as well - any clues on how it could be implemented ?

**Comment by benkeen** [comment]
Hey @jmls, check out the `scroll_locking` branch for some clues. I wasn't at all happy with the code, but the approach made sense I think. Hopefully that'll give you some clues.

**Comment by thanhdx0** [comment]
No update until now?

**Comment by eevleevs** [comment]
Not sure if related, but I get the two editors to scroll together by 
```js
var left = diff.editors.left.ace
var right = diff.editors.right.ace

$('.acediff__left .ace_scrollbar-v').scroll(() =>
    right.session.setScrollTop(left.session.getScrollTop()))
$('.acediff__right .ace_scrollbar-v').scroll(() =>
    left.session.setScrollTop(right.session.getScrollTop()))
```

**Comment by jmthomas** [comment]
If someone wants to implement this I think you could respond to [ScrollbarEvents](https://ajaxorg.github.io/ace-api-docs/interfaces/ace.Ace.ScrollbarEvents.html) and then send the same event to the other editor.
